### PR TITLE
fix(slug): Fix build with pinyin slugified

### PR DESCRIPTION
**Pull request by GreatWizard**

### CODE REVIEW

File: tests/unit/utils/string-test.js [hunk 1]
```diff
@@ -12,6 +12,6 @@ test('tests on slugify', (assert)=> {
   assert.equal(slugify(''), '');
   assert.equal(slugify('bonjour'), 'bonjour');
   assert.equal(slugify('annyǒng hashimnikka'), 'annyong-hashimnikka');
-  assert.equal(slugify('Bon jOuR Töi 1337   '), 'bon-jour-toi-1337');
+  assert.equal(slugify('Bon jOuR Töi 1337   '), 'bon-j-ou-r-toi-1337');
```
Comment (xcambar) [inline tests/unit/utils/string-test.js]: well... no. You _know_ the result is supposed to be `bon-jour-toi-1337`, don't you?

Comment (GreatWizard) [inline tests/unit/utils/string-test.js]: @xcambar We already ahve this discussion :p
I know that on a uppercase I split, Ember too... so it's correct

**Comment by GreatWizard** [comment]
I'm ok with that :)
I update it

But it sounds quite correct for me that: ShouldBeHere => should-be-here no?

**Comment by xcambar** [comment]
Do we want words to be split by uppercase letter?

**Pull request merged**